conf/bleedingEdge.neon

```diff
@@ -3,3 +3,5 @@ parameters:
 		bleedingEdge: true
 		skipCheckGenericClasses!: []
 		stricterFunctionMap: true
+		narrowPhpVersionFromComposerJson: true
```

Comment:
You have based this PR on 2.0.x. You shouldn't be adding a new flag to bleedingEdge.

Reply:
my thinking was, that this feature was never part of any stable 1.x release and therefore could need some time to stablize within bleeding edge.

dropped the toggle now